lib/parallel_cucumber/worker_manager.rb

```diff
@@ -67,7 +67,11 @@ def start_workers
         puts "Starting W#{index}"
         @workers["W#{index}"].start(env_for_worker(@options[:env_variables], index))
       end
-      @results.inject(:merge) # Returns hash of file:line to statuses + :worker-index to summary.
+      @results.inject do |seed, result|
+        seed.merge(result){|_key, oldval, newval|
+          (newval[:finish_time] > oldval[:finish_time]) ? newval : oldval
+        }
```

Comment:
formatting. let's use ```do .. end``` for multiline. But as this one is quite straightforward let's just make it a one-liner but add spaces around curly brackets.

Reply:
done